examples/streamz/Dockerfile

```diff
@@ -1,9 +1,12 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
 # An integration test & dev container which builds and installs CLX from default branch
 ARG RAPIDS_VERSION=0.16
```

Comment:
RAPIDS_VERSION should be updated to 0.17. Should also update `ci/release/update-version.sh` so that it's updated automatically with every new release.

Reply:
sure

Reply:
done